spec/system/admin/configuration/terms_of_service_files_spec.rb

```diff
@@ -21,20 +21,22 @@
 
     it "can be uploaded" do
       visit admin_terms_of_service_files_path
-      attach_file "Attachment", Rails.root.join(test_file_path)
-      click_button "Create Terms of service file"
+      attach_file "terms_of_service_file[attachment]", Rails.root.join(test_file_path)
```

Comment:
When you add the key to the label again then you can also use the label here:
```suggestion
      attach_file "Attachment", Rails.root.join(test_file_path)
```